src/types/types.ts

```diff
@@ -1,5 +1,34 @@
 import { IconName } from '@fortawesome/fontawesome-svg-core';
 
+export interface IPage {
+  id: string;
+  page_type: string;
+  title: string;
+  image: string;
+  introduction: {
+    copy: string[]
+  },
+  content: {
+    about: null | {
+      copy: string[]
+    },
+    info_pages: null | {
+      copy: string[],
+      title: string
+    },
+    collections: null | {
+      copy: string[],
+      title: string
+    },
+    introduction: null | {
+      copy: string[]
+    },
+  },
+  parent: [],
+  children: [],
+  collections: []
```

Comment:
collections has been replaced with collection_categories and collection_personas

Reply:
Amended